contracts/HydraDelegation/modules/VestedDelegation/IVestedDelegation.sol

```diff
@@ -49,6 +49,14 @@ interface IVestedDelegation is IDelegation {
         uint256 balanceChangeIndex
     ) external view returns (uint256);
 
+    /**
+     * @notice Calculates the delegators's generated rewards that are unclaimed
+     * @param staker Address of validator
+     * @param delegator Address of delegator
+     * @return reward Unclaimed rewards expected by the delegator from a staker (in HYDRA wei)
+     */
+    function calculateExpectedPositionReward(address staker, address delegator) external view returns (uint256 reward);
```

Comment:
Is there such a way for the vested staking positions to check their expected rewards as well?